src/util/translation.h

```diff
@@ -39,4 +39,9 @@ inline static bilingual_str _(const char* psz)
     return bilingual_str{psz, G_TRANSLATION_FUN ? (G_TRANSLATION_FUN)(psz) : psz};
 }
 
+inline static bilingual_str DoNotTranslate(std::string original)
```

Comment:
in commit aa21c807af55f94867b11bab5e2e9a8160a3fc9d

style-question: Could this be called `Untranslated`, so that arbitrary strings can be passed in (even string that are impossible to translate, such as error strings returned from third party libraries).

Reply:
Done.